src/app/shared/loggerHelper.js

```diff
@@ -38,4 +39,24 @@ module.exports = {
     }
     req.log.info({ message, level: "INFO", requestId: req.id });
   },
+  getMiddlewareErrorHandlerMessage: (err, description) => {
+    const { config, request, response, ...desiredErrorProperties } = err;
+
+    const requestDataString = config?.data;
+
+    const credentialIssuerId =
+      requestDataString && JSON.parse(requestDataString)?.credentialIssuerId;
```

Comment:
Do we know what would happen if JSON.parse() blew up (if `requestDataString` was invalid json for some reason)? Maybe wrap it to be super safe?

Reply:
It throws an error which is caught in the general error handling on the server